playwright/tests/integration/properties/input.spec.ts

```diff
@@ -66,3 +66,26 @@ test('cmsQuickfix', async ({ page }) => {
   await label.expectValue('Firstname');
   await label.expectInputValue("#{ivy.cms.co('/Labels/Firstname')}");
 });
+
+test('cmsToolTip', async ({ page }) => {
+  const editor = await FormEditor.openNewForm(page, { block: 'Input' });
+  await editor.canvas.blockByNth(0).inscribe();
+  await editor.inscription.expectHeader('Input');
+  const properties = editor.inscription.section('Properties');
+  const section = properties.collapsible('General');
+  const label = section.input({ label: 'Label' });
+  const badge = label.outputLocator.locator('.ui-flex').first();
```

Comment:
fine for me. but we should keep in mind that we should maybe add a badge pageobject if we need to test this a second time...